msicreator/createmsi.py

```diff
@@ -40,7 +45,6 @@ def __init__(self, jsonfile):
         self.version = jsondata['version']
         self.comments = jsondata['comments']
         self.installdir = jsondata['installdir']
-        self.license_file = jsondata['license_file']
```

Comment:
deleted 2 lines that created conflict with the license file.
(lines 48 and 382 in the original script)